features/google/src/model/analyses/utils/analysis-tools.js

```diff
@@ -7,3 +7,26 @@ export function groupAnalysisByKey(analysisEntries, keyGroup) {
     });
     return groupedAnalysisByKey;
 }
+
+export function convertFileSizeUnit(byteLength) {
```

Comment:
This is probably needed somewhere else, and I seem to remember something like this was already in the facebookImport, but have been unable to find it. It's bound to be useful elsewhere, so maybe consider it (later) for inclusion in `poly-analysis`